### ISSUE TITLE
Error raised in initial-setup.sh Script

### ISSUE THREAD

**Issue by marcboudreau** [Cloudshock/bootstrap#12]
When running the **initial-setup.sh** script, the following error occurs:
```
docker: Error response from daemon: create ./tmpKqlQ8Q: "./tmpKqlQ8Q" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
See 'docker run --help'.
```

**Comment by marcboudreau** [comment]
The error occurs because the **initial-setup.sh** script creates a temporary directory using a relative (`./`) path.  Directories cannot be volume mounted using relative paths.

**Comment by marcboudreau** [comment]
This issue was resolved by Pull Request #13.
